src/freenet/node/NewPacketFormat.java

```diff
@@ -169,7 +169,7 @@ List<byte[]> handleDecryptedPacket(NPFPacket packet, SessionKey sessionKey) {
 		
 		boolean dontAck = false;
 		boolean wakeUp = false;
-		if(packet.getError() || (packet.getFragments().size() == 0)) {
+		if(packet.getError() || (packet.getFragments().isEmpty())) {
```

Comment:
Parens.

Reply:
Fixed in ed98d8c127320b71b507f1524cce60e0d5efa356